tools/autograd/gen_annotated_fn_args.py

```diff
@@ -0,0 +1,29 @@
+from .utils import write, CodeTemplate
+from .gen_python_functions import get_py_nn_functions, get_py_torch_functions, op_name
+import textwrap
+from .gen_autograd import load_aten_declarations
+
+
+def gen_annotated(aten_path, out, template_path):
+    declarations = load_aten_declarations(aten_path)
+    annotated_args = []
+    for func in recurse_dict(get_py_torch_functions(declarations)):
+        annotated_args.append(process_func("torch._C._VariableFunctions", func))
+
+    for func in recurse_dict(get_py_nn_functions(declarations)):
+        annotated_args.append(process_func("torch._C._nn", func))
+
+    annotated_args = textwrap.indent("\n".join(annotated_args), "    ")
+    env = {"annotated_args": annotated_args}
+    PY_ANNOTATED_ARGS = CodeTemplate.from_file(template_path + '/templates/annotated_fn_args.py')
+    write(out, 'annotated_fn_args.py', PY_ANNOTATED_ARGS, env)
```

Comment:
Can you please post example of what is generated by this?

Reply:
It is likely that I will ask you to export only exactly what you need for the test, rather the entirety of everything in `declarations`

Reply:
This is because anything we generate into this module, despite its private naming, will implicitly become part of the public API, and the narrower we make this API, the less likely people will (1) start depending on it and (2) depend on it in a way that prevents us from making other changes we need to do

Reply:
I only exported what I need for the test.